To: vim_dev@googlegroups.com
Subject: Patch 7.3.357
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.357
Problem:    Compiler warning in MS-Windows console build.
Solution:   Adjust return type of PrintHookProc(). (Mike Williams)
Files:	    src/os_mswin.c


*** ../vim-7.3.356/src/os_mswin.c	2011-09-21 20:09:38.000000000 +0200
--- src/os_mswin.c	2011-11-30 13:41:13.000000000 +0100
***************
*** 1869,1875 ****
  
  #ifndef FEAT_GUI
  
!     static UINT CALLBACK
  PrintHookProc(
  	HWND hDlg,	// handle to dialog box
  	UINT uiMsg,	// message identifier
--- 1869,1875 ----
  
  #ifndef FEAT_GUI
  
!     static UINT_PTR CALLBACK
  PrintHookProc(
  	HWND hDlg,	// handle to dialog box
  	UINT uiMsg,	// message identifier
*** ../vim-7.3.356/src/version.c	2011-11-30 13:03:24.000000000 +0100
--- src/version.c	2011-11-30 13:41:55.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     357,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
215. Your mouse-clicking forearm rivals Popeye's.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
